README.md

```diff
@@ -1,157 +1,207 @@
 # ROCmValidationSuite
-The ROCm Validation Suite (RVS) is a system validation and diagnostics tool for monitoring, stress testing, detecting and troubleshooting issues that affects the functionality and performance of AMD GPU(s) operating in a high-performance/AI/ML computing environment. RVS is enabled using the ROCm software stack on a compatible software and hardware platform.
 
+<<<<<<< HEAD
```

Comment:
Conflict markers have to removed. Right ?

Reply:
yes, sorry--you got in here quicker than I anticipated! :)